scilpy/image/volume_math.py

```diff
@@ -447,7 +481,8 @@ def multiplication(input_list, ref_img):
         Multiply multiple images together (danger of underflow and overflow)
     """
     _validate_length(input_list, 2, at_least=True)
-    _validate_imgs(*input_list, ref_img)
+    _validate_imgs_type(*input_list)
+    _validate_same_shape(*input_list, ref_img)
```

Comment:
do we really need to validate this here ? it should have been done in the script, no ?

Reply:
No, because in the script, we don't know which input is an image. It depends on the operation being called. That's why verifications are done in all sub-functions.